helpers/update-site/documentation.js

```diff
@@ -701,13 +705,19 @@ const createHintCategories = (files) => {
         if (!acc.has(category)) {
             acc.set(category, {
                 description: config.categories[category].description,
-                hints: [hintName],
+                hints: [{
+                    friendlyName: file.frontMatter.title,
```

Comment:
Is this pulling from `meta.ts` or `README.md` for the hint?

Reply:
I think this is pulling from `README.md`

Reply:
That is something that maybe we can think in change when we start using the packages to build the docs